charts/fluent-bit/values.yaml

```diff
@@ -44,13 +44,15 @@ podSecurityPolicy:
   create: false
   annotations: {}
 
-openShift:
-  # Sets Openshift support
+openshift:
+  # Enable Openshift support
   enabled: false
-  # Creates SCC for Fluent-bit when Openshift support is enabled
   securityContextConstraints:
-    create: true
+    # Create SCC for Fluent-bit when Openshift support is enabled
+    create: false
     annotations: {}
+    # Allow Fluent-bit to use existing SCC in cluster
+    name: "privileged"
```

Comment:
```suggestion
    name:
```
I don't think this value should be set if it's intended to be user defined.

Reply:
In this case I should make ClusterRole condition a bit different to prevent unexpected behavior:
- If "name" isn't empty use it
- else create predefined SCC and use it's name in ClusterRole

Are you ok with it?

Reply:
I'd suggest either copying the service account creation logic which would allow the end-user to choose the name of the SCC the chart creates or keeping it simple as you suggest and using the name to decide if a SCC needs creating.

Reply:
@stevehipwell 
One of the features I've wanted to implement is to make it possible to use predefined SCC in cluster. OpenShift has a bunch of SCCs installed in cluster out of the box, which could be more strict than one I've added to this Chart. It's suitable for the cases when Fluent-Bit doesn't require of hostPath volumes and privileged container. 

To cover both cases, I advertise you such way of implementation:
```
# OpenShift-specific configuration
openShift:
  securityContextConstraints:
    # Create SCC for Fluent-bit when Openshift support is enabled
    create: false
    annotations: {}
    # Provide name for SecurityContextConstraints will be created
    # or use one existing with the name provided
    name: ""
```

It will cause following possible configurations:
- .Values.openShift.securityContextConstraints.create=true and no name provided: create SCC from template named by-default and allow to use it
- .Values.openShift.securityContextConstraints.create=true and the name provided: create SCC from template named as user wants and allow to use it
- .Values.openShift.securityContextConstraints.create=false and no name provided: do not create SCC, do not allow to use any of SCCs
- .Values.openShift.securityContextConstraints.create=false and no name provided: do not create SCC, but allow to use SCCs with  .Values.openShift.securityContextConstraints.name

How are you with such implementation?

Reply:
I'm OK with it, but what's about ClusterRole permissions precedence?  
If ```create: true``` and ```existingName``` isn't empty, what SCC Fluent-Bit will allowed to use? Both?